src/frontend/src/components/NavBar/components/AppNavBar/components/Announcements/index.tsx

```diff
@@ -1,39 +1,33 @@
-import { useSelector } from "react-redux";
-import { selectCurrentPathogen } from "src/common/redux/selectors";
 import { B } from "src/common/styles/basicStyle";
-import { SplitPathogenWrapper } from "src/components/Split/SplitPathogenWrapper";
-import { PATHOGEN_FEATURE_FLAGS } from "src/components/Split/types";
+import { USER_FEATURE_FLAGS } from "src/components/Split/types";
 import { StyledBanner, StyledNewTabLink } from "./style";
+import { useTreatments } from "@splitsoftware/splitio-react";
+import { isUserFlagOn } from "src/components/Split";
 
 // Note: this was previously for privacy policy announcements
 // It was pretty convenient to reuse - maybe we want to keep
 // it around for one-off banners.
 
-// Show Banner on Covid pages to let the user know that we
-// are currently unable to ingest GISAID data
+// Show Banner to let users know we are sunsetting the app
 export const Announcements = (): JSX.Element => {
-  const pathogen = useSelector(selectCurrentPathogen);
+  const flag = useTreatments([USER_FEATURE_FLAGS.sunset_banner]);
+
+  const shouldShowSunsetBanner = isUserFlagOn(
+    flag,
+    USER_FEATURE_FLAGS.sunset_banner
+  );
 
   return (
     <>
-      <SplitPathogenWrapper
-        pathogen={pathogen}
-        feature={PATHOGEN_FEATURE_FLAGS.show_gisaid_ingestion_banner}
-      >
+      {shouldShowSunsetBanner && (
         <StyledBanner sdsType="primary">
-          <B>
-            As of 4/13/23, we have switched from using GISAID for phylo tree
-            focal and contextual samples to using GenBank.&nbsp;
-          </B>
-          <StyledNewTabLink
-            href={
-              "https://help.czgenepi.org/hc/en-us/articles/14696829683860-GISAID-to-GenBank-Transition"
-            }
-          >
-            Learn more
+          <B>CZ GEN EPI will shut down on March 7, 2024. Click&nbsp;</B>
+          <StyledNewTabLink href={"https://help.czgenepi.org/hc/en-us"}>
```

Comment:
This link needs to be updated once it's ready: https://czi-sci.slack.com/archives/C05RZ888CF3/p1697480548561429

Reply:
done!